### ISSUE TITLE
[BasicUI] Extended default settings for BasicUI sitemaps

### ISSUE THREAD

**Issue by multimartin** [openhab/openhab-webui#2594]
In OH 4.2.x the settings for BasicUI most settings for the sitemaps appearance has been moved to the settings "wheel" on the sitemap page - and stored in the browser.
As a user of sitemaps from headless systems also, where I call and render html-pages from a shell command - I have no possibility to set and store the settings for that client.
Would it be possible to keep all general settings possibilities in the main web UI for BasicUI sitemaps - whilst still keeping the option to override those default settings in the individual browser?

So basically add more default options in the main UI - similar to how it could be set pre-4.2.x? Right now you can only enable/disable inline SVG and set the name of the default sitemap.

**Comment by lolodomo** [comment]
Having same settings at two different places will confuse everybody and makes no sense IMHO.

> As a user of sitemaps from headless systems also, where I call and render html-pages with a shell command - I have no possibility to set and store the settings for such a client.

This is a very unusual way to use UI I guess. For my curiosity, what is your usage exactly ?
One option could be to add new optional query parameters to the Basic UI URL (one for each setting) that will overwrite the browse settings.
